downstream/modules/platform/proc-aap-hub-restore.adoc

```diff
@@ -0,0 +1,30 @@
+[id="aap-hub-restore"]
+
+= Recovering the {HubNameStart} deployment
+
+[role=_abstract]
+Use this procedure to restore a previous hub deployment into the namespace. The deployment name you provide will be the name of the new AutomationHub custom resource that will be created.
+
+[NOTE]
+====
+The name specified for the new AutomationHub custom resource must not match an existing deployment or the recovery process will fail.
+====
+
+.Prerequisites
+
+* You must be authenticated with an Openshift cluster.
+* The {ControllerName} has been deployed to the cluster.
```

Comment:
This should be `{HubName}` not Controller.

Reply:
Fixed.